src/util/localStorage.ts

```diff
@@ -0,0 +1,24 @@
+import _ from 'lodash'
+import { IStorage } from '../types'
+
+let storage: IStorage = {
```

Comment:
Let's name this file `storage.ts`.

We have a convention of naming the file after the default export when possible.